src/licensedcode/data/rules/false-positive_788.RULE

```diff
@@ -0,0 +1,6 @@
+---
+is_false_positive: yes
+notes: haha
+---
+
+license[sizeof(license) - 1] = 0;
```

Comment:
```suggestion
license[sizeof(license)
```

Reply:
Not sure why, but changing this to the shorter false positive rule is not removing this detection, but the full rules are removing them allright